sssd_test_framework/roles/client.py

```diff
@@ -94,3 +94,18 @@ def setup(self) -> None:
                     raise ValueError("List is not expected")
 
                 self.sssd.import_domain(domain, role)
+
+    def set_sssd_user(self, user: str) -> bool:
```

Comment:
Can you move it to SSSDUtils? This way, it will be also usable on the IPA server, not just the client.